scripts/util.cjs

```diff
@@ -398,11 +398,49 @@ const generateMainnetTokenList = async (fullTokenList) => {
   return outputTokenList
 }
 
+const generateDappLists = async () => {
+  const dappRadarProjectId = process.env.DAPP_RADAR_PROJECT_ID
+  const dappRadarApiKey = process.env.DAPP_RADAR_API_KEY
+  const chains = [
+    'solana',
+    'ethereum',
+    'polygon',
+    'binance-smart-chain',
+    'optimism',
+    'aurora',
+    'avalanche',
+    'fantom',
+  ]
+  const metric = 'uaw'
+  const range = '30d'
+  const top = 100
+  const dappLists = {}
+
+  for (const chain of chains) {
+    const url = `https://api.dappradar.com/${dappRadarProjectId}/dapps/top/${metric}?chain=${chain}&range=${range}&top=${top}`
+    const response = await fetch(url, {
+      headers: {
+        'X-BLOBR-KEY': dappRadarApiKey,
+      },
+    })
+
+    if (response.ok) {
+      const dapps = await response.json()
+      dappLists[chain] = dapps
+    } else {
+      console.error(`Error fetching dApps for ${chain}: ${response.status} ${response.statusText}`)
```

Comment:
It looks like we return an empty object in case of an error? If not the case already, we should halt execution of the script in order to prevent publishing bad data inadvertently.

Reply:
I updated for existing `fetch` calls too.